pkgs/node/src/clock.zig

```diff
@@ -29,14 +33,7 @@ pub const Clock = struct {
         const current_slot = @divFloor(@as(isize, @intCast(std.time.milliTimestamp())) + CLOCK_DISPARITY_MS - genesis_time_ms, SECONDS_PER_SLOT_MS);
         const current_slot_time_ms = genesis_time_ms + current_slot * SECONDS_PER_SLOT_MS;
 
-        return Self{
-            .genesis_time_ms = genesis_time_ms,
-            .current_slot_time_ms = current_slot_time_ms,
-            .current_slot = current_slot,
-            .events = events,
-            .timer = timer,
-            .c = c,
-        };
+        return Self{ .genesis_time_ms = genesis_time_ms, .current_slot_time_ms = current_slot_time_ms, .current_slot = current_slot, .events = events, .timer = timer, .c = c, .on_slot_cbs = std.ArrayList(*const fn (ud: ?*anyopaque, slot: isize) void).init(allocator) };
```

Comment:
autoformat? it's less readable this way

Reply:
yea how to do it into multiline? wanted to ask you this (it auto formats)

Reply:
you need to add a comma after the last entry. I'll merge and push the fix on master.